build/setup.sql

```diff
@@ -81,49 +81,50 @@ insert into lead_providers(id, name) values
   (3, 'TF')
 ;
 
-create table tenureships (
+create table rosters (
   id integer primary key,
   participant_id integer references participants(id),
   school_id integer references schools(id),
+  role varchar(64),
   started_on date not null,
   finished_on date null
 );
 
-insert into tenureships(id, participant_id, school_id, started_on, finished_on) values
+insert into rosters(id, participant_id, school_id, role, started_on, finished_on) values
 
   -- ects
-  (1, 1, 1, '2021-09-01', null),
-  (2, 2, 1, '2021-09-02', '2022-09-02'),
-  (3, 3, 1, '2021-09-03', null),
-  (4, 4, 1, '2021-09-04', null),
-  (5, 5, 1, '2021-09-05', null),
-  (6, 6, 1, '2021-09-04', null),
-  (7, 7, 1, '2021-09-05', null),
-  (8, 8, 2, '2021-09-06', null),
-
-  (9, 9, 2, '2022-09-01', null),
-  (10, 10, 2, '2022-09-02', null),
-  (11, 11, 2, '2022-09-03', null),
-  (12, 12, 2, '2022-09-04', null),
-  (13, 13, 2, '2022-09-05', '2022-12-04'),
+  (1, 1, 1, 'ect', '2021-09-01', null),
+  (2, 2, 1, 'ect', '2021-09-02', '2022-09-02'),
+  (3, 3, 1, 'ect', '2021-09-03', null),
+  (4, 4, 1, 'ect', '2021-09-04', null),
+  (5, 5, 1, 'ect', '2021-09-05', null),
+  (6, 6, 1, 'ect', '2021-09-04', null),
+  (7, 7, 1, 'ect', '2021-09-05', null),
+  (8, 8, 2, 'ect', '2021-09-06', null),
+
+  (9, 9, 2, 'ect', '2022-09-01', null),
+  (10, 10, 2, 'ect', '2022-09-02', null),
+  (11, 11, 2, 'ect', '2022-09-03', null),
+  (12, 12, 2, 'ect', '2022-09-04', null),
+  (13, 13, 2, 'ect', '2022-09-05', '2022-12-04'),
 
   -- mentors
-  (20, 20, 1, '2021-09-05', '2022-09-05'),
-  (21, 21, 1, '2021-09-05', null),
-  (22, 22, 1, '2021-09-05', null),
-  (23, 23, 1, '2021-09-05', null),
+  (20, 20, 1, 'mentor', '2021-09-05', '2022-09-05'),
```

Comment:
I was wondering about storing the role too. Can't we work out if someone's an ECT or Mentor by checking whether or not they're mentoring or being mentored?

It felt to me like holding an indicator on whether they are considered able to be a mentor makes sense (so we can add them to the pool).

Reply:
It feels a bit dangerous to assume that a mentor will _always_ be mentoring, probably worth exploring though whether something like that could work but I worry about inferring too much from other attributes as that has caught us out before

Reply:
If we have participants rostered at a school in a role (and rostered multiple times if necessary) then we don't need a mentor pool or anything separate any more, the roster provides that

Reply:
Also gives us one place to look for all the participants at a school and what role that are performing